homeassistant/components/webostv/__init__.py

```diff
@@ -98,55 +157,61 @@ async def async_service_handler(service):
             DOMAIN, service, async_service_handler, schema=schema
         )
 
-    tasks = [async_setup_tv(hass, config, conf) for conf in config[DOMAIN]]
-    if tasks:
-        await asyncio.gather(*tasks)
+    hass.data[DOMAIN][host] = {"client": client}
```

Comment:
Usually we store these indexed by entry id.  I think you are doing this for the notify platform though